.github/workflows/deploy-v2.yml

```diff
@@ -0,0 +1,72 @@
+name: Deploy v2
+concurrency: deploy_v2_${{ github.ref_name }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: Environment to deploy to
+        required: true
+        type: choice
+        options:
+        - review_aks
+      pr-number:
+        description: PR number for the review app
+        required: false
+      sha:
+        description: Commit sha to be deployed
+        required: true
+
+permissions:
+  contents: write
+  deployments: write
+  packages: write
+  pull-requests: write
+
+jobs:
+  deploy-v2-review-app:
+    name: Deployment To Review v2
+    concurrency: deploy_v2_review_${{ inputs.pr-number }}
+    # if: ${{ github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'deploy_v2') }}
+    # needs: [build]
```

Comment:
Leftovers?

Reply:
This needs rework anyway as it is review app specific at the moment. I'll rework this when we're happy to merge.